registry.json

```diff
@@ -15,6 +15,18 @@
         "operatorName": "Michael Van Stan",		                                   
         "github": "MichaelVanStan"      
     },
+    {
+        "ticker": "LYNX",                                                      
+        "poolId": "5e759d13110d744637b618668b54b30b12cb638bb3c5400654338c54",
+        "social": {                                                            
+            "website": "https://cryptolynxadapool.com",
+            "instagram": "lynx_cardano_stake_pool",  
+            "twitter": "lynx_pool"
+        },
+        "telegram": "LYNX_STAKEPOOL",
```

Comment:
Please change this to be the telegram handle of a user and not a channel or group